src/components/project_view/submission_detail/ag_test_command_result_detail.vue

```diff
@@ -33,6 +33,8 @@
                         {'actual-return-code-correct': ag_test_command_result !== null
                           && ag_test_command_result.return_code_correct,
                           'actual-return-code-incorrect': ag_test_command_result !== null
+                          // return_code_correct can be null so we need to check for false
```

Comment:
Hmm why is it showing up as a new change now? I see this comment on develop after pulling locally...

Reply:
Oops. I think this also came from the other PR. I have now deleted the comment commit from this branch :)